sample/settings.gradle

```diff
@@ -13,6 +13,9 @@ pluginManagement {
     maven { url 'https://www.jetbrains.com/intellij-repository/releases' }
     maven { url "https://cache-redirector.jetbrains.com/intellij-dependencies" }
     maven { url "../build/localMaven" }
+    maven {
+      url 'https://oss.sonatype.org/content/repositories/snapshots/'
```

Comment:
im moving sql-psi out of the fatjar and making it a normal dependency, because otherwise it gets minimized and APIs that the dialects use that the gradle plugin does not disappear and you end up with classpath errors during compilation

i have chosen a very dark path indeed

Reply:
(this wont get merged to master, its only on the sql-psi-dev branch)